redis/src/main/scala/zio/redis/RedisExecutor.scala

```diff
@@ -25,7 +26,7 @@ object RedisExecutor {
 
   private[this] final val RequestQueueSize = 16
 
-  private[this] final val StreamedExecutor =
+  final val StreamedExecutor =
```

Comment:
Made this public so I can use it in the second executor, maybe not a good idea

Reply:
I would keep it hidden.